src/components/Tools/AdvisorCard.vue

```diff
@@ -85,25 +87,29 @@ export default defineComponent({
     color: #3d3d3d;
     align-self: center;
     justify-self: start;
-    margin-left: 0.25rem;
-    margin-right: 0.25rem;
+    //margin-left: 0.25rem;
+    //margin-right: 0.25rem;
```

Comment:
Can we delete this styling if it's unused

Reply:
lol yup i forgot… that was part of my experimentation